src/commands/createNewProject/pythonSteps/IPythonVenvWizardContext.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "vscode-azureextensionui";
+
+export interface IPythonVenvWizardContext extends IActionContext {
+    projectPath: string;
+    pythonAlias?: string;
```

Comment:
How about a setting to set this by default so that users can skip the python alias prompt if they want?

Reply:
Will wait for user feedback. We already have a few venv/python related settings and I don't feel like adding to the [confusion](https://github.com/microsoft/vscode-azurefunctions/issues/1393)